dora/examples/pom.xml

```diff
@@ -50,7 +50,6 @@
     <dependency>
       <groupId>org.alluxio</groupId>
       <artifactId>alluxio-shaded-client</artifactId>
-      <version>2.9.0</version>
```

Comment:
This mistakenly depends on a specific version.

Reply:
this one was intentional, see https://github.com/Alluxio/alluxio/pull/16647#discussion_r1050568979